docs/visualization/README.md

```diff
@@ -0,0 +1,38 @@
+# Support for Visualization tab
+
+The following visualizations have been tested:
+
+- Pie Chart
+- Vertical Bars
+- Area
+- Horizontal Bars
+- Gauge
+- Data Table
+- Heat Map
+- Goal
+- Metric
+
+## Bucket Aggregations
+
+The following bucket aggregations ("Buckets" in the Kibana UI) have been implemented:
+
+- Date Histogram
+- [Date Range](date_range.md)
+- [Filters](filters.md)
+- Histogram
+- [Range](range.md)
+- [Terms](terms.md)
+
+## Metric Aggregations
+
+The following metric aggregations ("Metrics" in the Kibana UI) have been implemented:
+
+- Average
+- Count
+- Max
+- Median
+- Min
+- Percentiles
+- Standard Deviation
+- Sum
+- Unique Count
```

Comment:
Also add: Top Hits (will be merged this week)

Reply:
Done